spec/lib/mvi/service_spec.rb

```diff
@@ -131,6 +64,26 @@
           expect(response.profile).to have_deep_attributes(mvi_profile)
         end
       end
+
+      it 'fetches historical icns if they exist' do
+        allow(user).to receive(:mhv_icn).and_return('1008714701V416111^NI^200M^USVHA^P')
+
+        VCR.use_cassette('mvi/find_candidate/historical_icns_with_icn') do
```

Comment:
can you change these new tests to use `VCR::MATCH_EVERYTHING`? it should just work if you stub `allow(SecureRandom).to receive(:uuid).and_return('5e819d17-ce9b-4860-929e-f9062836ebd0')` and set timecop to `'Wed, 21 Feb 2018 20:19:01 GMT'`